marketplace/commercelayer-sku/src/index.css

```diff
@@ -0,0 +1,202 @@
+body {
+    overflow: hidden;
+    font-family: "Avenir Next W01", Helvetica, sans-serif;
+}
+
+.name {
+    min-height: 72px;
+    font-weight: 600;
+}
+
+.sku,
+.selected-sku {
+    min-height: 350px;
+    border: 1px solid #eeeeee;
+    margin: 10px;
+    padding: 20px;
+}
+
+.sku {
+    cursor: pointer;
+}
+
+.sku:hover {
+    border: 1px solid #b4c3ca;
+}
+
+.pagination {
+    margin: 5px;
+}
+
+.cf-form-field {
+  margin-bottom: 1.928571428571429em;
+}
+.cf-form-field > label {
+  display: block;
+  margin-bottom: 0.642857142857143em;
+}
+
+input.cf-form-input {
+  font-size: 16px;
+  color: #2a3039;
+  font-family: "Avenir Next W01", Helvetica, sans-serif;
+  display: block;
+  width: 100%;
+  line-height: 1.5;
+  padding: 0 2px;
+  background-color: transparent;
+  border-width: 0 0 1px 0;
+  border-style: solid;
+  border-color: transparent;
+  border-image-width: 0 0 1px 0;
+  border-image-source: url("https://static.contentful.com/app/svg/dotted-border.svg");
+  border-image-repeat: round;
+  border-image-slice: 1 1;
+  -webkit-transition: 0.2s linear;
```

Comment:
@suevalov Is `contentful-extension-scripts` set-up for autoprefixer? If not, should we add it? Would be nice to do this automatically.

Reply:
Good catch @realityking It can be easily enabled cause we use Parcel under the hood, but it is not enabled at the moment. I'll create an issue for that.

Reply:
https://github.com/contentful/create-contentful-extension/issues/43

Reply:
Is #43 going to block merging this extension?

Reply:
nope

Reply:
👍🏿 I'm going to resolve this comment then for now.